apps/server/lib/bootstrap.ts

```diff
@@ -99,14 +103,14 @@ const SCHEMA_ACTIVE_TYPE_CONTRACTS: JSONSchema = {
 };
 
 const getActorKey = async (
-	context: coreType.Context,
-	jellyfish: coreType.JellyfishKernel,
+	logContext: LogContext,
+	jellyfish: Kernel,
```

Comment:
Rename this to `kernel`. It doesn't reference the whole Jellyfish system of anything like that.

Reply:
Sure thing. Will do. Regarding the casts to `any`, they should be temporary. We can get rid of them once we bump the queue module as well.

Reply:
@edufschmidt no, bump both instead. Otherwise you'd be passing a v8 `Kernel` to the queue that expects a v11 `Kernel`. There are some incompatible changes.